src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,44 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <div className="card" data-cy="Movie" key={movie.imdbId}>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
+          alt="Film logo"
```

Comment:
Consider adding a descriptive alt attribute, for example, use the title of the film here